src/dis6/CMakeLists.txt

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
I am not completely opposed to glob, but I prefer being explicit. because its easy to get into weird dependency overlap, which is what seems to be happening with the utils directory, which I think needs to be avoided in order to clarify having the dis6 and dis7 examples.

Reply:
Fair point. In that case, it is probably better to leave it as explicit.